src/client/eppo-precomputed-client.ts

```diff
@@ -51,32 +62,36 @@ export default class EppoPrecomputedClient {
   private precomputedFlagsRequestParameters?: PrecomputedFlagsRequestParameters;
   private subjectKey?: string;
   private subjectAttributes?: Attributes;
+  private decodedFlagKeySalt = '';
 
   constructor(
     private precomputedFlagStore: IConfigurationStore<PrecomputedFlag>,
-    private isObfuscated = false,
+    private isObfuscated = true,
```

Comment:
Made the default true because we expected all use cases of the precomputed client to be using an obfuscated response format

Reply:
Non-obfuscated format is not defined, so there's no use for this flag
```suggestion
```

Also, `obfuscated` flag is embedded in configuration itself, so the client should respect it instead of out-of-band `isObfuscated` flag that is easy to mess up